### PR TITLE
Fb master fancy login

### LINKED ISSUE #11: Fix routing in application
For example opening localhost:5100/my-area generates a 404. It should instead redirect to the my-area.

### LINKED ISSUE #13: Put footer at bottom if there is few content
When there is not a lot of content, the footer (on desktop) is in the middle of the screen. It would be better to place the footer at the bottom of the page. However, if the content is quite high, the footer must be appended just like normal.

### PR DESCRIPTION
**Pull request by svenpopping**
Fixes #13
Fixes #11

### CODE REVIEW

File: app/elements/elements.html [hunk 1]
```diff
@@ -24,6 +23,8 @@
 <link rel="import" href="lancie-login/lancie-login-form.html">
 <link rel="import" href="lancie-login/lancie-ajax.html">
 
+<link rel="import" href="lancie-ticket-page/lancie-ticket-page.html">
+
```
Comment (TimvdLippe) [inline app/elements/elements.html]: Why was this import moved?

File: app/elements/lancie-ticket-page/lancie-ticket-block.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
+<link rel="import" href="/bower_components/polymer/polymer.html">
+
+<dom-module id="lancie-ticket-block">
+  <link rel="import" type="css" href="lancie-ticket-block.css">
+
+  <template>
+    <paper-material elevation="0" class="ticket-selector-block layout vertical" style$="{{_computeStyle(color)}}">
+      <div class="version layout horizontal self-end"><span>{{ticket.numberSold}}</span>/<span>{{ticket.limit}}</span></div>
+      <h2>{{ticket.ticketType}}</h2>
+      <h3 class="horizontal layout start">Price: € <span>{{price}}</span></h3>
```
Comment (TimvdLippe) [inline app/elements/lancie-ticket-page/lancie-ticket-block.html]: The spans can be removed as Polymer supports compound bindings

File: app/elements/lancie-ticket-page/lancie-ticket-block.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          return 'background-color: ' + color;
+        },
+
+        updatePrice: function() {
+          var basePrice = this.ticket.price;
+
+          basePrice += this.chMember ? this.ticket.discountPrice : 0;
+          basePrice += this.pickup ? this.ticket.pickupService : 0;
+
+          this.price = basePrice;
```
Comment (TimvdLippe) [inline app/elements/lancie-ticket-page/lancie-ticket-block.html]: I would change this to a computed property instead. So you can remove the observers from the properties and remove the assignment.

File: app/elements/lancie-ticket-page/lancie-ticket-block.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+          },
+
+          price: {
+            type: String,
+            value: function() {
+              return 0;
+            }
+          },
+
+          notAvailable: {
```
Comment (TimvdLippe) [inline app/elements/lancie-ticket-page/lancie-ticket-block.html]: Maybe a better name is "disabled"? As it disables the buttons and checkbox

File: app/styles/main.css [hunk 5]
```diff
@@ -14,6 +14,7 @@ body {
 }
 
 iron-pages {
+  min-height: calc(100vh - 64px);
```
Comment (TimvdLippe) [inline app/styles/main.css]: I think this fixes the min height issue, but please create a separate PR for that.

**Comment by TimvdLippe** [comment]
The branch must be rebased on current master. Also there are quite some elements added which I will take a closer look at tomorrow.

File: app/elements/lancie-login/lancie-login-form.html [hunk 6]
```diff
@@ -32,9 +69,10 @@
       headers="{{headerFrom('application/json', csrfToken)}}"
       on-response="onLoggedIn"></iron-ajax>
 
-    <paper-card heading="{{getHeading(registering)}}">
+    <paper-card heading="{{getHeading(registering)}}" image="http://placehold.it/300x100" class="white">
```
Comment (skamoen) [inline app/elements/lancie-login/lancie-login-form.html]: Maybe replace this with the A5L logo before merging into master?

**Comment by TimvdLippe** [comment]
LGTM

**Comment by svenpopping** [comment]
Merge ALL THE THINGS
